utils/synthetics/getMarketsOpenInterest.ts

```diff
@@ -39,10 +40,9 @@ export async function getMarketsOpenInterest(
     acc[market.marketToken] = marketCalls
     return acc
   }, {})
-
-  const results = (await client.multicall({
-    contracts: Object.values(callsByMarket).flat(),
-  })) as { result: bigint; status: string; error?: string }[]
+  
+  const allCalls = Object.values(callsByMarket).flat()
+  const results = await batchedMulticall<{ result: bigint; status: string; error?: string}>(client, allCalls, 30)
```

Comment:
if one if internal multicall requests fail then some items in results would be undefined. but there is no checks for undefined inside `reduce`'s callback